server/tokenapi/AnimeList.go

```diff
@@ -0,0 +1,35 @@
+package tokenapi
+
+import (
+	"errors"
+
+	"github.com/animenotifier/notify.moe/arn"
+)
+
+type AnimeParameters struct {
+	AnimeName    string
```

Comment:
If you use the prefix "Anime" on all of the fields and also indicate that this is anime related via the struct name, that field prefix is most likely redundant.